hammer_time/hammer_time.py

```diff
@@ -52,6 +78,19 @@ def perform(client, host_id):
         remove_and_wait(client, sorted(new_cont))
 
 
+class KillMongoDAction:
+    """Action to kill mongod."""
+
+    def generate_parameters(client):
+        return {'machine_id': choose_machine(client.get_controller_client())}
+
+    def perform(client, machine_id):
+        ctrl_client = client.get_controller_client()
+        ctrl_client.juju('ssh', (machine_id, 'sudo', 'pkill', 'mongod'))
+        # The impact of killing mongod seems to be delayed.
+        time.sleep(5)
```

Comment:
Once juju notices that mongodb is dead, there is an obvious impact, because when you pkill mongodb, it defaults to SIGTERM, and systemd does not restart this service.  SIGTERM and a few other signals are treated as graceful termination.

I do not know at this time why the impact is not immediate, but it's definitely not immediate.

I am tweaking this to wait until "pgrep mongod" fails before returning, instead of sleeping for 5 seconds.